app/views/landing-pages/league/PageLeague.vue

```diff
@@ -20,7 +20,6 @@
 
       created () {
         this.fetchRequiredInitialData({ optionalIdOrSlug: this.$route.params.idOrSlug })
-        this.loadGlobalRequiredData()
```

Comment:
Accidentally always loading the global leaderboard even when looking at a clan page.